main/dede.cc

```diff
@@ -22,7 +22,9 @@
 #include "deepdetect.h"
 #include "commandlineapi.h"
```

Comment:
According to #559 these should be controlled with #ifdef as well, no ? cc @fantes